glusterd2/volume/store-utils.go

```diff
@@ -189,14 +189,18 @@ func GetAllBricksInCluster() ([]brick.Brickinfo, error) {
 }
 
 // AreReplicateVolumesRunning retrieves the volinfo objects from GetVolumes() function
-// and checks if all replicate, Disperse volumes are stopped before
-// stopping the self heal daemon.
-func AreReplicateVolumesRunning() (bool, error) {
+// and checks if all replicate, Disperse volumes are stopped except input volume ID
+//before stopping the self heal daemon.
```

Comment:
* needs space after `//` for these two line 193 and 194
* s/Disperse/disperse

The comment can be improved as follows:

```
AreReplicateVolumesRunning checks if all replicate and disperse volumes are stopped.
The volume being acted upon is excluded from this check and the volume ID of that volume
needs to be volume passed as an argument.
```